test/perf/react.js

```diff
@@ -5,17 +5,17 @@ const { React, ReactDOM } = window;
 
 
 // Skate components.
-const wclist = (props, chren) => ul(props, chren);
-const wcitem = (props, chren) => li(props, chren);
-const [ div, h1, li, ul, item, list ] = ['div', 'h1', 'li', 'ul', wclist, wcitem].map(t => vdom.element.bind(null, t));
+const wclist = (ps, chren) => ul(ps, chren); // eslint-disable-line no-use-before-define
+const wcitem = (ps, chren) => li(ps, chren); // eslint-disable-line no-use-before-define
```

Comment:
That's something I missed in my original review, nice catch. Line 10 there shouldn't have been removed, I think.